cluster-scope/overlays/prod/common/kustomization.yaml

```diff
@@ -43,5 +43,6 @@ resources:
     - ../../../base/user.openshift.io/groups/thoth
     - ../../../base/user.openshift.io/groups/thoth-devops
     - ../../../base/user.openshift.io/groups/uky-redhat
+    - ../../../base/user.openshift.io/groups/ushift-dev
```

Comment:
CI is failing because this group doesn't exist. The group is named `octo-ushift-dev`.